great-idea/index.html

```diff
@@ -14,10 +14,81 @@
   <![endif]-->
 </head>
 
-  <body>
-
-    <!-- Copy and paste your HTML from the first UI project here -->
-      
+<body class="main-container">
+    <header>
+      <nav>
+        <a href="services.html">Services </a>
+        <a href="#">Product </a>
+        <a href="#">Vision </a>
+        <a href="#">Features </a>
+        <a href="#">About </a>
+        <a href="#">Contact </a>
+      </nav>
+      <div>
+        <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
+      </div>
+    </header>
+    
+    <section > 
+      <div class="section-text">
+        <h2  class="first">
+          Innovation 
+        </h2>
+        <h2  class="second">
+          On
+        </h2>
+        <h2  class="third">
+          Demand
+        </h2>
+        <p>Get Started</p>
+      </div>
+    
+    <div  class="image">
+      <img  src="img/header-img.png" alt="Image of a code snippet.">
+    </div>
+  </section>
+    <div class="br"></div>
```

Comment:
I see you made a div and gave it a br class to make the break.  There's also a `<br>` tag that might have made your life a bit easier